packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -612,8 +613,36 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
     this.addEventListener('copy', this.handleCopy);
     this.addEventListener('cut', this.handleCut);
     this.addEventListener('paste', this.handlePaste);
+    this.addEventListener('selectionChanged', this.handleSelectionChanged);
   }
 
+  /**
+   * Disable the inner input of the fluent-text-field.
+   */
+  private disableTextInput() {
+    const inputControl = this.input.shadowRoot.querySelector<HTMLInputElement>('input');
+    inputControl?.setAttribute('disabled', 'true');
+    inputControl.value = '';
```

Comment:
this should probably be using an optional chaining just incase inputControl doesn't exist

Reply:
Yes. I'm putting it in an if block because you can't assign to an optional value.